src/scripts/generateChangelog.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  generateMarkDown,
+  parseCommits,
+  getGitDiff,
+  getLastGitTag,
+  getCurrentGitTag,
+  loadChangelogConfig,
+} from "changelogen";
+import ChangelogenConfig from "../../changelog.config";
+
+
+async function generateChangelog() {
+  const config = await loadChangelogConfig(process.cwd(), {
+	...ChangelogenConfig,
+	from: getCurrentGitTag()
+  });
+
+  const rawGitCommits = await getGitDiff(
+	getCurrentGitTag(),
+	await getLastGitTag()
+  );
+  const newCommits = parseCommits(rawGitCommits, config);
+
+  /**
+   * Generate the changelog content
+   * @returns {string} The changelog in markdown format
```

Comment:
TSDoc doesn't use the `{}` syntax.

Reply:
Fixed.

Reply:
Your JSDoc was good, you should've kept the @returns. Just omit "`{string} `".

Reply:
> Your JSDoc was good, you should've kept the @returns. Just omit "`{string} `".

It is actually redundant

Reply:
> Your JSDoc was good, you should've kept the @returns. Just omit "`{string} `".

It is actually redundant

Reply:
Knowing that it returns it as markdown isn't redundant, knowing that it's a string when the signature tells you that is.

Reply:
> Knowing that it returns markdown isn't redundant, telling the reader it's a string when they know it from the signature is.

I use `@returns` in functions because it makes more sense because I call the function and it **returns** a value

In our case, the variable is a constant, so I don't see the point of using `@returns`.

Reply:
Oh, yep. I didn't notice they were variables 🤦‍♂️